package.json

```diff
@@ -31,21 +31,23 @@
     "typechain": "cross-env TS_NODE_TRANSPILE_ONLY=true hardhat typechain",
     "build": "rm -rf dist && hardhat compile && hardhat compile --config hardhat.config.zksync.ts && tsc --declaration",
     "docgen": "hardhat docgen",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "yarn patch-package"
   },
   "dependencies": {
     "@chainlink/contracts": "^0.5.1",
-    "@defi-wonderland/smock": "2.3.5",
+    "@defi-wonderland/smock": "2.4.0",
     "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
     "@openzeppelin/contracts": "^4.6.0",
     "@openzeppelin/contracts-upgradeable": "^4.7.3",
     "@venusprotocol/governance-contracts": "^2.4.0",
     "@venusprotocol/solidity-utilities": "^2.0.0",
     "@venusprotocol/venus-protocol": "^9.1.0",
     "ethers": "^5.6.8",
-    "hardhat": "2.19.5",
+    "hardhat": "2.22.15",
```

Comment:
I think hardhat 2.22.18 supports solidity 0.8.28, which supports the `transient` keyword, and it generates the associated bytecode. That would make the code cleaner

Reply:
https://github.com/NomicFoundation/hardhat/commit/e47b495ec1cbb7ea63e8a2639607b350cb732ce1

Reply:
Yes, but our contracts are using `0.8.25` version and already deployed and we cannot upgrade the contracts with `0.8.28` version.